<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="commStatButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>commStatButton</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="targetLocatorTextEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="targetLocatorLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Target Locator:</string>
    </property>
   </widget>
   <widget class="QLabel" name="searcherMoverLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Searcher Mover:</string>
    </property>
   </widget>
   <widget class="QLabel" name="pathFindingLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>PathFinder:</string>
    </property>
   </widget>
   <widget class="QLabel" name="targetGrabberLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Target Grabber</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="searcherMoverTextEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="pathFinderTextEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="targetGrabberTextEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="sentCommStatTextEdit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sentCommStatLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sent commStat:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="connectToServerCheckBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect to Server</string>
    </property>
   </widget>
   <widget class="QLabel" name="connectedToServerLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect to Server:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="connectedToServerTextEdit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>497</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBasicGuiClientTCP">
    <property name="title">
     <string>BasicGuiClientTCP</string>
    </property>
   </widget>
   <addaction name="menuBasicGuiClientTCP"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
